packages/spor-react/src/alert/BaseAlert.tsx

```diff
@@ -3,7 +3,13 @@ import React from "react";
 
 export type BaseAlertProps = BoxProps & {
   /** The color scheme and icon of the alert */
-  variant: "info" | "success" | "warning" | "alt-transport" | "error";
+  variant:
+    | "info"
+    | "success"
+    | "warning"
+    | "alt-transport"
+    | "error"
+    | "service";
```

Comment:
Brukes denne enda hvis dere har laget en ny komponent?

Reply:
Denne brukes for å få riktig bakgrunnsfarge fordi ServiceAlert bruker BaseAlert-komponenten, og Per Øyvind sa at han ønsket den "grønne med fill-ikon"-varianten på vanlig alert/expandable alert også.